src/version-check.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs')
+var chalk = require('chalk')
+var latestVersion = require('latest-version')
+
+module.exports = function () {
+  latestVersion('dat').then(function (version) {
+    fs.writeFile('../.dat-version', version, function (err) {
```

Comment:
We want to write to a in the home-dir (or somewhere that'll be the same no matter where Dat is run). Maybe write to `~/.dat/cli-version.txt` (we use os-home module elsewhere)?